src/plugins/mini/mini.c

```diff
@@ -60,6 +60,8 @@ static inline char * findUnescapedEquals (char * text)
 	{
 		before = equals++;
 	}
+	/* The function will return a pointer to the `\0` character at the end of the string
+	   if it did not find any unescaped `=` character. */
```

Comment:
Thank you, looks nice! Please put it above the function. Please also document (via asserts) preconditions (`text != 0`).